### ISSUE TITLE
Use job to get full file from chunks

### ISSUE THREAD

**Issue by lucabecchetti** [pionl/laravel-chunk-upload#131]
I, is there a way to dispatch a job to combine all chuncks asynchronously?

**Comment by pionl** [comment]
At this moment no.

**Comment by pionl** [comment]
How would you expect API response?

**Comment by chandlerwilcox88** [comment]
I have this same need. For my app a simple success response is all that's necessary.

**Comment by dokicro** [comment]
I need this too, since Cloudflare timeouts after 100seconds

**Comment by bvicini** [comment]
I agree it would be a useful feature. Joining chunks is really time consuming and timeouts happen for very large files on some servers.

**Comment by abourtnik** [comment]
Any news for this issue ?
